conf/distro/include/sourcery-tuning.inc

```diff
@@ -16,6 +16,6 @@ TUNE_CCARGS_append = "${@bb.utils.contains('TUNE_FEATURES', 'ppce500mc', ' ' + d
 
 PPCE6500_CCARG = "${@'-te6500' if d.getVar('SOURCERY_GXX_IS_PRO', True) == '1' else '-mcpu=e6500'}"
 TUNE_CCARGS_append = " ${@bb.utils.contains("TUNE_FEATURES", "e6500", ' ' + d.getVar('PPCE6500_CCARG', True), '', d)}"
-TUNE_ASARGS += "${@bb.utils.contains("TUNE_FEATURES", "m64", "-a64 -me6500", "", d)}"
+TUNE_ASARGS_e6500-64b += "${@bb.utils.contains("TUNE_FEATURES", "m64", "-a64 -me6500", "", d)}"
```

Comment:
I think it'd have been cleaner to mirror what TUNE_CCARGS does, using the e6500 flag, rather than this, but that's minor, and this'll do.